.github/workflows/auto-labeling.yml

```diff
@@ -0,0 +1,14 @@
+on:
+  schedule:
+    # Run this action every 10 minutes
+    - cron: "*/10 * * * *"
+
+jobs:
+  labeling:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: harupy/auto-labeling@master
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          label-pattern: '- \[(.*?)\] ?`(.+?)`'
```

Comment:
What this action does is:

1. Fetch issues and pull requests (created within a month past)
2. For each issue or pull request, extract labels from the description and add (or remove) them.

Reply:
@harupy awesome! Do you have an idea of how many Github REST API requests the action makes? Is it proportional to the number of issues & PRs created within the past month? Would just be good to sanity-check that we won't run into the [REST API limits on Github actions](https://docs.github.com/en/actions/getting-started-with-github-actions/about-github-actions#usage-limits):

> You can execute up to 1000 API requests in an hour across all actions within a repository. If exceeded, additional API calls will fail, which might cause jobs to fail. This limit also applies to self-hosted runners.

Reply:
If it seems like we'll hit the threshold, maybe we can run the action less frequently, e.g. every hour?

Reply:
It is proportional to the number of issues & PRs & labels that need to be updated. Given the threshold, every-10-minutes is too frequent. I'll fix it to every-hour as you suggested.